packages/fxa-settings/src/lib/canvas-utils.tsx

```diff
@@ -29,11 +29,11 @@ export async function getCroppedImg(
   const canvas = document.createElement('canvas');
   const ctx = canvas.getContext('2d');
 
+  // Calculate a safe area to store image so that it isn't clipped during rotation by
+  // canvas context. The 1.5 multiplier was found during manual testing of various
+  // photos on default iOS.
   const maxSize = Math.max(image.width, image.height);
-  const safeArea = 2 * ((maxSize / 2) * Math.sqrt(2));
-
-  // set each dimensions to double largest dimension to allow for a safe area for the
-  // image to rotate in without being clipped by canvas context
+  const safeArea = 1.5 * ((maxSize / 2) * Math.sqrt(2));
```

Comment:
hmm, 1.5 probably buys us some more headroom than 2 but since (afaict) `maxSize` is based on the source image dimensions it doesn't seem like this fixes the issue completely. 

Since we have a fixed final size with `pixelCrop` we ought to be able to set the initial canvas size to that and use the resizing variant of [drawImage](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/drawImage). _(I haven't tried this)_

Reply:
Sure, let me give that a try 👍🏽

Reply:
I can still get this to black out on the simulator. I've got an ugly code sketch that fixes it... _i think_